src/main/java/com/lob/model/USAutocompletion.java

```diff
@@ -0,0 +1,137 @@
+package com.lob.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.lob.exception.APIException;
+import com.lob.exception.AuthenticationException;
+import com.lob.exception.InvalidRequestException;
+import com.lob.exception.RateLimitException;
+import com.lob.net.APIResource;
+import com.lob.net.LobResponse;
+import com.lob.net.RequestOptions;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class USAutocompletion extends APIResource {
+    public static final String RESOURCE = "us_autocompletions";
+
+    public static class Suggestion {
```

Comment:
An alternative would be to change the package name to be `com.lob.model.usverifications` (java packages shouldn't have underscores), and then put all your public sub classes inside USAutocompletion into their own files so they don't have to be static.